src/server/string_family_test.cc

```diff
@@ -108,6 +108,22 @@ TEST_F(StringFamilyTest, Keepttl) {
   EXPECT_TRUE(actual > 0 && actual <= 200);
 }
 
+TEST_F(StringFamilyTest, GetSet) {
+  string old_value = "old_value";
+  string new_value = "new_value";
+  auto args_old_value = RespExpr::buffer(&old_value);
```

Comment:
I do not think you need to introduce `args_old_value` and `args_new_value`.
afaik, `ASSERT_EQ(res_old_value, old_value)` should work for line 123.